README.md

```diff
@@ -392,7 +392,7 @@ instance = SingleVMOpenEdXInstance.objects.create(
     sub_domain='dogwood.sandbox',
     name='Dogwood',
     fork_name='edx/edx-platform',
-    branch_name='named-release/dogwood',
+    branch_name='named-release/dogwood.rc',
```

Comment:
@itsjeyd Damn, sorry, I'm stupid here - I had missed that in your comment. That makes everything simpler. : ) Thanks, and sorry for the trouble.